agriculture/data/product_pricelist.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_pricelist_2" model="product.pricelist">
+        <field name="name">B2B</field>
+        <field name="selectable" eval="True"/>
+    </record> 
```

Comment:
only pricelist? any rules or pricelist item?

Reply:
Yes only pricelist and no rules or pricelist item

Reply:
Yes only pricelist